backend/services/apple_auth_service.py

```
@@ -0,0 +1,154 @@
+import jwt  # PyJWT package
+import requests
+import time
+import base64
+import os
+from typing import Dict, Optional
+from cryptography.hazmat.primitives import serialization
```

Ruff (0.11.9)

`cryptography.hazmat.primitives.serialization` imported but unused

Remove unused import: `cryptography.hazmat.primitives.serialization`

(F401)